src/OpenLineageWriter.php

```diff
@@ -33,15 +35,23 @@ public function write(): void
             }
             $this->logger->info(sprintf('Job %s import to OpenLineage API - start', $job['id']));
 
-            $lineAgeResponse = $this->getJobLineage($job['id']);
-            foreach ($lineAgeResponse as $event) {
+            $lineageResponse = $this->getJobLineage($job['id']);
+            foreach ($lineageResponse as $event) {
                 if ($this->jobNameAsConfig) {
                     $event['job']['name'] = sprintf('%s-%s', $job['component'], $job['config']);
                 }
                 $this->logger->info(sprintf('- Sending %s event', $event['eventType']));
-                $this->openLineageClient->request('POST', '/api/v1/lineage', [
-                    'body' => json_encode($event),
-                ]);
+
+                try {
+                    $this->openLineageClient->request('POST', '/api/v1/lineage', [
+                        'body' => json_encode($event),
+                    ]);
+                } catch (RequestException $e) {
+                    if (str_contains($e->getMessage(), 'cURL error 3:')) {
```

Comment:
je to nutný to řešit separátně?

Reply:
Jako neni, ale k tomu curl 3 je ta hlaska debilna

Reply:
Curl 3:
`cURL error 3:  (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)`

Curl 6:
`cURL error 6: Could not resolve host: wrong-api (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)`

Tak preto no

Reply:
V pripade potreby tam mozme doplnit dalsie ify

Reply:
Jo na to jsem se ptal, jestli to jako nenapíše vůbec nic nebo něco použitelnýho, takže to nenapíše vůbec nic